bower.json

```diff
@@ -42,6 +44,11 @@
     "spice-html5-bower": "himdel/spice-html5-bower#^0.1.5",
     "es6-shim": "^0.35.0",
     "phantomjs-polyfill": "^0.0.2",
-    "fetch": "^1.0.0"
+    "fetch": "^1.0.0",
+    "rxjs": "^4.1.0",
+    "rx-angular": "rx.angular#^1.1.3"
+  },
+  "resolutions": {
```

Comment:
Aah, yes, it does happen from time to time .. but in our scripts we call `bower install -F` just to make it not ask.. 

So unless it breaks stuff for you, can you still remove it and resolve it only locally? So that it's used only for resolutions that aren't like "meh, whatever works"...

Anyway, really looking forward to `npm` :)